site/content/deployment/enmasse.md

```diff
@@ -0,0 +1,56 @@
++++
+title = "OpenShift"
+weight = 475
++++
+
+## Build Hono
+
+~~~sh
+~/hono$ mvn clean install -Pbuild-docker-image
+~~~
+
+## Deploy EnMasse
+
+Dowload the release, unpack and run
+
+~~~sh
+./deploy-openshift.sh -n hono
+~~~
+
+## Deploy Hono
+
+~~~sh
+~hono/example/target/deploy/openshift$ chmod +x *
+~hono/example/target/deploy/openshift$ ./enmasse_deploy.sh
+~~~
+
+## Extract Certificate
+
+~~~sh
+~hono/example$ oc extract secret/external-certs-messaging --to=target/config/hono-demo-certs-jar/ -n hono
```

Comment:
what is this certificate good/required for?
Is this the `server-cert.pem` that is referenced in the *Start Consumer* command line?

Reply:
This is the certificate needed by AMQP clients for connecting to EnMasse (the messaging AMQP endpoint) for starting to receive messages (referring to the Hono architecture).

Reply:
Yes. It's for the clients connecting to the enmasse (from outside of openshift)